__tests__/app.test.js

```diff
@@ -44,3 +44,30 @@ describe("GET /api/", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  it("200: should respond with an article with the correct id and all expected properties", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toHaveLength(1);
+        expect(body[0]).toHaveProperty("author");
+        expect(body[0]).toHaveProperty("title");
+        expect(body[0]).toHaveProperty("article_id");
+        expect(body[0]).toHaveProperty("body");
+        expect(body[0]).toHaveProperty("topic");
+        expect(body[0]).toHaveProperty("created_at");
+        expect(body[0]).toHaveProperty("votes");
+        expect(body[0]).toHaveProperty("article_img_url");
+      });
```

Comment:
Nice use of `toHaveProperty` here - though it may be easier to achieve the same thing using `toMatchObject`.
One thing to consider is - how would your test know you got the correct article back, and not, for example, article 3? 🤔 
It would strengthen the test to check the retrieved article contains the right content, and has the expected id. ✅